### DISCUSSION TITLE
Getting application id inside the application

### DISCUSSION THREAD

**Discussion by jimdaro** [Budibase/budibase#5650] [Help]
Hi Folks,

Is it possible to get the app_id by Handlebars or Javascript as a variable?

Thx

**Comment by melohagan** [comment]
Hi @jimdaro 

I don't think this binding is available. What is your use case?

**Comment by jimdaro** [comment]
I'm using Make (formerly Integromat) as middle system, and I'd like to write back data to ceratin budibase instances but I don't want to hardcode it into the system - if the Budibase system could tell its id then I could find all the details with api and write the data back.
